index.html

```diff
@@ -75,6 +75,7 @@ <h2 id="project_tagline">GitHub入门与实践</h2>
 <p>您如果有对本书的读后感，或者只是想单纯测试一下，都可以发送Pull Request。</p>
 
 <p>(现共有<span id="impression_num"></span>条用户来稿)</p>
+<p class="impression">第一次使用GitHub，第一次感受开源的魅力。还有，makedown太牛逼了！！！(<a href="https://github.com/shadot">@shadot</a>)</p>
```

Comment:
呃呃，是的markdown~  感谢您的指出。书很棒
----- 原始邮件 -----
发件人：Bin Liu <notifications@github.com>
收件人：ituring/first-pr <first-pr@noreply.github.com>
抄送人：Shadot Nan <sunshine_ny@sina.com>,  Author <author@noreply.github.com>
主题：Re: [ituring/first-pr] Add my impression @shadot (#373)
日期：2016年08月01日 11点51分

In index.html:
> @@ -75,6 +75,7 @@ <h2 id="project_tagline">GitHub入门与实践</h2>
>  <p>您如果有对本书的读后感，或者只是想单纯测试一下，都可以发送Pull Request。</p>
>  
>  <p>(现共有<span id="impression_num"></span>条用户来稿)</p>
> +<p class="impression">第一次使用GitHub，第一次感受开源的魅力。还有，makedown太牛逼了！！！(<a href="https://github.com/shadot">@shadot</a>)</p>

markdown?

&mdash;
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.